### DISCUSSION TITLE
Media wiki schema example

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-orm#944] [Show and tell]
https://codeberg.org/LifeSymbiont/media_wiki/src/commit/870d13dc48af66d348b4795088883f0ced60a8f1
![](https://lemmy.ml/pictrs/image/592d7759-7f00-4c59-a38e-87e4e974dce5.png)

**Comment by billy1624** [comment]
Hey @JediMaster25, welcome! What can I help? :D

**Reply by ghost** [reply]
Just leaving an example of usage of sea-orm. I've already completed it.

**Reply by billy1624** [reply]
Alright! Thanks for the updates :)

**Comment by tyt2y3** [comment]
Nice. It's really a non-trivial schema! 

https://codeberg.org/LifeSymbiont/media_wiki/src/commit/870d13dc48af66d348b4795088883f0ced60a8f1/src/migrator/m20220726_000001_create_schema.rs

You can also add it to the `Community.md` once you got an initial release!
